package.json

```diff
@@ -67,14 +67,12 @@
         "chai": "~4.2.0",
         "chai-as-promised": "~7.1.1",
         "chai-string": "~1.5.0",
-        "eslint": "^6.8.0",

```

Comment:
Eslint is the standard for Node.js and TypeScript in particular. It does a lot more than just formatting - it's primarily for preventing silly mistakes in code (like not awaiting a promise). I would highly recommend re-enabling eslint ASAP or never removing it in the first place.

Can you quickly try using v7 instead of v6 and see if that works? v7 should get rid of the npm audit complaints and you can turn off any failing rules in [here](https://github.com/Azure/azure-functions-durable-js/blob/dev/.eslintrc) instead of disabling _all_ of eslint. I do think it's fine to get rid of "prettier-eslint", though, because I don't think you're using it. Feel free to ping me if you have issues

Reply:
I'll reach out internally on Monday and block this merge for a little bit. Thanks for reaching out!

Reply:
No problem, just subscribed to the repo like last week 😁 👀

![always watching](https://media.giphy.com/media/flWn1Cyk9HcVIj9xWy/giphy.gif)

Reply:
@ejizba: I managed to get this working by upgrading `eslint` and its related dependencies. Could I please get another review? Thanks